docs/src/reference/the-traversal.asciidoc

```diff
@@ -2185,6 +2185,28 @@ g.V().out().out().path().by(
                                  __.in('created').values('name')).fold())
 ----
 
+It's possible to limit the path using the <<to-step,`to()`>> or <<from-step,`from()`>> step modulators.
+
+[gremlin-groovy,modern]
+----
+g.V().has('person','name','vardas').as('e').
```

Comment:
thanks for the pull request. i was a bit surprised to see we had no examples for that. could you please these are good examples, though, you misspelled "vadas" as "vardas" so the queries return no results. if you could please fix that, i'd be happy to get this merged.

Reply:
oops, that's embarrassing. Changed.